### PR TITLE
Added ability to donate ETH

### PR DESCRIPTION
**Pull request by samelliott89**
### PR Checklist (check all)

* [x] Updated `CHANGELOG.md` to describe the included fixes and changes made
* [x] Included issue number on the description of the PR
* [x] Commented on the relevant issue thread about this PR

### Issue number

#200

### Changelog

Added ability to donate ETH

### CODE REVIEW

**Comment by ricburton** [comment]
Really excited to dive into this 🤜🤛

Cheers,

Richard

File: src/modals/DonationModal.js [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+    color ? `rgb(${colors[color]})` : `rgb(${colors.dark})`};
+`;
+
+const StyledFlex = styled.div`
+  display: flex;
+  position: relative;
+  transform: none;
+`;
+
+const StyledBottomModal = styled(StyledFlex)`
```
Comment (samelliott89) [inline src/modals/DonationModal.js:66]: @pedrouid if you can think on how to better share styles (StyledActions, StyledApproveTransaction etc) and other pieces across the two modals apart from going ahead and making a lot of them components...? Very copy paste at the moment...

File: src/helpers/utilities.js [hunk 2]
```diff
@@ -143,3 +143,19 @@ export const bootIntercom = () => {
...
+ */
+export const getEth = assets => {
+  return assets.filter(asset => asset.symbol === 'ETH')[0];
+};
+
+/**
+ * @desc returns default eth address for donations
+ * @return {String}
+ */
+export const balanceManagerEthAddress =
```
Comment (samelliott89) [inline src/helpers/utilities.js:160]: Also not sure if you want this in a build config?

Comment (pedrouid) [reply src/helpers/utilities.js:160]: Will probably move out of the repo into a env variable

Comment (samelliott89) [reply src/helpers/utilities.js:160]: Sure, I couldn't see a config file... Lmk if you want me to whip one up.

File: src/modals/DonationModal.js (hunk 1 above)
Comment (pedrouid) [reply src/modals/DonationModal.js:66]: Totally agree, will be working on that

**Comment by pedrouid** [comment]
@samelliott89 Made some final changes, mostly design changes and moved the address to a server config.  Thanks again! 🙌

**Pull request merged**